scripts/hid_configurator/configurator_cli.py

```diff
@@ -186,13 +196,11 @@ def perform_led_stream(dev, args):
 
 
 def parse_arguments():
-    try:
```

Comment:
removed parser = argparse.ArgumentParser() - as compliance did not allow it.
We require usage of python => 3.9  so I think we can do it

Reply:
I will double check this and get back to you